src/test/java/persistence/study/ReflectionTest.java

```diff
@@ -0,0 +1,90 @@
+package persistence.study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ReflectionTest {
+    private static final Logger logger = LoggerFactory.getLogger(ReflectionTest.class);
+
+    @DisplayName("Car 객체 정보 가져오기")
+    @Test
+    void showClass() {
+        final Class<Car> carClass = Car.class;
+
+        Arrays.stream(carClass.getDeclaredFields()).forEach(System.out::println);
+        Arrays.stream(carClass.getConstructors()).forEach(System.out::println);
+        Arrays.stream(carClass.getDeclaredMethods()).forEach(System.out::println);
+    }
+
+    @DisplayName("test로 시작하는 메소드 실행한다.")
+    @Test
+    void testMethodRun() throws NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
+        final Class<Car> carClass = Car.class;
+        final Car car = carClass.getDeclaredConstructor().newInstance();
+
+        for (Method method : carClass.getDeclaredMethods()) {
+            if (method.getName().startsWith("test")) {
+                final String result = (String) method.invoke(car);
+                logger.info(result);
+            }
+        }
+    }
+
+    @DisplayName("@PrintView 애노테이션 메소드 실행")
+    @Test
+    void testAnnotationMethodRun() throws NoSuchMethodException, InvocationTargetException, InstantiationException, IllegalAccessException {
+        final Class<Car> carClass = Car.class;
+        final Car car = carClass.getConstructor().newInstance();
+
+        for (Method method : carClass.getDeclaredMethods()) {
+            if (method.isAnnotationPresent(PrintView.class)) {
+                method.invoke(car);
+            }
+        }
```

Comment:
#### 객체지향 생활 체조 원칙
> 규칙 1: 한 메서드에 오직 한 단계의 들여쓰기만 한다.

Reply:
뎁스를 줄이기 위해서 고민해보면 어떨까요? 
다양한 방법이 있을것 같아요.

ex)
```suggestion
        Arrays.stream(carClass.getDeclaredMethods())
            .filter(method -> method.isAnnotationPresent(PrintView.class))
            ...
```

Reply:
넵 반영하겠습니다.